pkgs/data/fonts/smc-manjari/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenvNoCC, fetchurl, truetype ? false }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "smc-manjari";
+  version = "2.200";
+
+  src = fetchurl {
+    url = "https://gitlab.com/api/v4/projects/490109/jobs/artifacts/Version${version}/raw/build/manjari-Version${version}.tar.gz?job=build-tag";
```

Comment:
I've seen Gitlab job artifacts disappear before, but I'm not sure what the policy on gitlab.com is. Probably okay to leave as is.

Reply:
would it be better to switch to building from source? 

upstream seems to be working on improving their CI: https://gitlab.com/smc/fonts/manjari/-/merge_requests/15
not sure if this will change the artifact url in the future

Reply:
Oh, thanks for linking that MR. It seems they are planning to upload artifacts to the "Gitlab registry". I'm not familiar with it, but I hope the build artifacts will be available for a long time there.

I see two options for this PR:

1. translate their build instructions into the Nix package
2. use the job artifact for now + the official tarball later

Option 2 is easier and less likely to break with future updates.

Reply:
#264555 is also a font from the same org. both of these have similar build process. They both have a Makefile that can build the font files and only depends on one python dependency for it.

In that one, I am building it from the source because the job artifacts for the latest version was not available

Reply:
Ah I see. I didn't really notice the two packages were that similar. In that case I think it would be best to build it from source in both packages (i.e. copy the approach from #264555 to this PR).

Reply:
Updated. now it's being built from source